.pre-commit-config.yaml

```diff
@@ -19,13 +19,13 @@ repos:
       - id: isort
         args: ["--profile", "black"]
   - repo: https://github.com/PyCQA/docformatter
-    rev: master
+    rev: v1.7.5
```

Comment:
> - [github.com/PyCQA/docformatter: master → v1.7.5](https://github.com/PyCQA/docformatter/compare/master...v1.7.5)

@stavros11 it seems it will try to update on its own (in reference to https://github.com/qiboteam/qibolab/pull/1081#issuecomment-2441518384).

Not only: apparently we'll have to fight to keep `master` 🤦